ingest/ingest_pipeline.py

```diff
@@ -91,7 +91,9 @@
 
 class IngestPipeline(object):
     # File location for metadata json convention
-    JSON_CONVENTION = 'gs://broad-singlecellportal-public/AMC_v1.1.3.json'
+    JSON_CONVENTION = (
+        '../schema/alexandria_convention/alexandria_convention_schema.json'
```

Comment:
Blocker: 

`JSON_CONVENTION` is used [in our public CLI](https://github.com/broadinstitute/single_cell_portal/blob/d39c79533c126efc1b825e5263aae5d5a67696c6/scripts/manage_study.py#L132).  To not break that, this must be a GS URL to a publicly-readable file, e.g. gs://broad-singlecellportal-public/alexandria_convention_schema.json`.

Also, albeit not quite blocking to me: we should consider keeping a version string somewhere in that URL.

Reply:
created EXTERNAL_JSON_CONVENTION `f204f50`
will require corresponding change in [the public CLI](https://github.com/broadinstitute/single_cell_portal/blob/d39c79533c126efc1b825e5263aae5d5a67696c6/scripts/manage_study.py#L132)

Reply:
Thanks for this --  dependency management isn't easy!